lib/_debugger.js

```diff
@@ -1588,7 +1588,8 @@ Interface.prototype.repl = function() {
   this.repl.on('exit', exitDebugRepl);
 
   // Set new
-  this.repl.eval = this.debugEval.bind(this);
+  this.repl.eval = (code, ctx, file, cb) =>
+     this.debugEval(code, ctx, file, cb);
```

Comment:
Seems our lint rules are lax with arrow functions. I see indentations of 1, 3 and 6 in this PR when breaking after `=>`. I'd suggest treating it as if it were a normal `function() {` and indent the next line 2 spaces. Don't want to hold up the PR on this, but also think it would be good to start off w/ consistency since this'll be the first commit with heavy introduction of arrow function usage. :)